Update cloud-cli-installation.md (#5088)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/configure-cloud-cli.md

pagination_next: null
---

> Find out how to configure the dbt Cloud CLI for your dbt Cloud project. Doing so enables you to run dbt commands, like `dbt environment show` to view your dbt Cloud configuration or `dbt compile` to compile your project and validate models and tests. You'll also benefit from:

Reviewer: ```suggestion
Learn how to configure the dbt Cloud CLI for your dbt Cloud project to run dbt commands, like `dbt environment show` to view your dbt Cloud configuration or `dbt compile` to compile your project and validate models and tests. You'll also benefit from:
```

Once you install the dbt Cloud CLI, you need to configure it to connect to a dbt Cloud project. 

> 1. Go to dbt Cloud to download your `dbt_cloud.yml` credentials file by navigating to **Develop** and select **Configure dbt Cloud CLI**.

Reviewer: ```suggestion
1. In dbt Cloud, navigate to **Develop** and click **Configure dbt Cloud CLI** to download your `dbt_cloud.yml` credentials file.
```

- `dbt --help`: Lists the commands available for dbt<br />
- `dbt run --help`: Lists the flags available for the `run` command
:::

## FAQs
<expandable alt_header="How to create a .dbt directory and move your file">

> If you've never had a `.dbt` directory, you should perform the following recommended steps to create one. If you already have `.dbt` directory, then you should be able to move the `dbt_cloud.yml` file into it.

Reviewer: ```suggestion
If you've never had a `.dbt` directory, you should perform the following recommended steps to create one. If you already have a `.dbt` directory, move the `dbt_cloud.yml` file into it.
```

<Tabs>
<TabItem value="Create a .dbt directory">

  1. Clone your dbt project repository locally.
  2. Use the `cd` command to change directories and navigate to the root of your filesystem. (For example, if you're a Mac user it might be `Macintosh HD -> Users -> YOUR_USERNAME`):

     ```bash
     cd ~/Users
      ```
  3. Use the `mkdir` command followed by the name of the folder you want to create. To create a `.dbt` folder, use:

     ```bash
     mkdir .dbt
     ```

This will create a `.dbt` folder in the current directory. 

> For Mac users, since it's a hidden folder (due to the dot prefix), it won't be visible in Finder by default. However, to view hidden files and folders press Command + Shift + G. 

Reviewer: ```suggestion
For Mac users, since it's a hidden folder (due to the dot prefix), it won't be visible in Finder by default. To view hidden files and folders, press Command + Shift + G. 
```

</TabItem>

<TabItem value="Move the dbt_cloud.yml file">

**On Mac or Linux**

> - In your command line, use the `mv` command to move your `dbt_cloud.yml` file into the `.dbt` directory. If you've just downloaded the `dbt_cloud.yml` file and it's in your Downloads folder, the command might look something like this:

Reviewer: We can probably move this line up next to **On Mac or Linux** with an &mdash; so it's not a floating single bullet point. Another possibility would be ### heading each of these rather than bolding them and having a broken-up list where the bullets are spread out under different lines. Per our style guide, bullets should be in a list format with more than one item in succession.

So it would look like: 

### Mac or Linux
In your command line, use the `mv` command to move your `dbt_cloud.yml` file into the `.dbt` directory. If you've just downloaded the `dbt_cloud.yml` file and it's in your Downloads folder, the command might look something like this:

```bash
mv ~/Downloads/dbt_cloud.yml ~/.dbt/dbt_cloud.yml
```

Author: i did have it as an h3 but thought it looked too bold. but I'll go with that suggestion! ty!

```bash
mv ~/Downloads/dbt_cloud.yml ~/.dbt/dbt_cloud.yml
```

**On Windows**
- In your command line, use the move command. Assuming your file is in the Downloads folder, the command might look like this:

```bash
move %USERPROFILE%\Downloads\dbt_cloud.yml %USERPROFILE%\.dbt\dbt_cloud.yml
```

</TabItem>
</Tabs>

> This command moves the `dbt_cloud.yml` from the Downloads folder to the `.dbt` folder. If your `dbt_cloud.yml` file is located elsewhere, adjust the path accordingly.

Reviewer: Why does the `.dbt` folder name get code formatted but not the Downloads folder name? Do we use code formatting to call attention to names?

Author: yea it was trying to make it clear since the .dbt folder is a hidden folder. could also format 'downloads' too
